backends/ebpf/targets/ebpfenv.py

```diff
@@ -16,155 +16,157 @@
     interfaces. The bridge runs in a completely isolated namespace.
     Allows the loading and testing of eBPF programs. """
 
-import os
 import sys
-from subprocess import Popen, PIPE
+import logging
+from pathlib import Path
 
-sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)) + '/../../tools')
-from testutils import *
+# Append tools to the import path.
+FILE_DIR = Path(__file__).resolve().parent
+# Append tools to the import path.
+sys.path.append(str(FILE_DIR.joinpath("../../../tools")))
+import testutils
 
 
-class Bridge(object):
+class Bridge:
 
-    def __init__(self, namespace, outputs, verbose):
-        self.ns_name = namespace  # identifier of the namespace
-        self.br_name = "core"  # name of the central bridge
-        self.br_ports = []  # list of the veth pair bridge ports
-        self.edge_ports = []  # list of the veth pair edge ports
-        self.outputs = outputs  # contains standard and error output
-        self.verbose = verbose  # do we want to be chatty?
+    def __init__(self, namespace):
+        self.ns_name = namespace # identifier of the namespace
+        self.br_name = "core"    # name of the central bridge
+        self.br_ports = []       # list of the veth pair bridge ports
+        self.edge_ports = []     # list of the veth pair edge ports
 
     def ns_init(self):
-        """ Initialize the namespace. """
-        cmd = "ip netns add %s" % self.ns_name
-        errmsg = "Failed to create namespace %s :" % self.ns_name
-        result = run_timeout(self.verbose, cmd, TIMEOUT, self.outputs, errmsg)
+        """Initialize the namespace."""
+        cmd = f"ip netns add {self.ns_name}"
+        errmsg = f"Failed to create namespace {self.ns_name} :"
+        result = testutils.exec_process(cmd, errmsg).returncode
```

Comment:
is the timeout now implicit in `exec_process`?

Reply:
Yes, the default value is 10 minutes.